tests/e2e/specs/backend/active-filters.test.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import {
+	insertBlock,
+	getEditedPostContent,
+	getAllBlocks,
+	switchUserToAdmin,
+	openDocumentSettingsSidebar,
+} from '@wordpress/e2e-test-utils';
+
+import {
+	visitBlockPage,
+	clearAndFillInput,
+} from '@woocommerce/blocks-test-utils';
+import { insertBlockDontWaitForInsertClose } from '../../utils';
+
+const block = {
+	name: 'Active Product Filters',
+	slug: 'woocommerce/active-filters',
+	class: '.wc-block-active-filters',
+};
+
+if ( process.env.WP_VERSION < 5.3 )
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( 'skipping all other things', () => {} );
+
+describe( `${ block.name } Block`, () => {
+	beforeAll( async () => {
+		await switchUserToAdmin();
+		await visitBlockPage( `${ block.name } Block` );
+	} );
+
+	it( 'can only be inserted once', async () => {
+		await insertBlockDontWaitForInsertClose( block.name );
+		expect( await getAllBlocks() ).toHaveLength( 1 );
+	} );
+
+	it( 'renders without crashing', async () => {
+		await expect( page ).toRenderBlock( block );
+	} );
+
+	it( "allows changing the block's title", async () => {
+		await openDocumentSettingsSidebar();
+		await clearAndFillInput(
+			'.wp-block[data-type="woocommerce/active-filters"] textarea.wc-block-editor-components-title',
+			'New Title'
+		);
+		await page.click(
+			'.components-toolbar button[aria-label="Heading 6"]'
+		);
+		await expect(
+			page
+		).toMatchElement(
+			'.wp-block[data-type="woocommerce/active-filters"] h6 textarea',
+			{ text: 'New Title' }
+		);
+
+		expect( await getEditedPostContent() ).toMatchSnapshot();
+		await clearAndFillInput(
+			'.wp-block[data-type="woocommerce/active-filters"] textarea.wc-block-editor-components-title',
+			'Active filters'
+		);
+		await page.click(
+			'.components-toolbar button[aria-label="Heading 3"]'
```

Comment:
It is not clear what the test is clicking here. Maybe a comment above could explain this step. Also, this would be like a good use case for searching something by text, as it is more human-friendly.

Also, Heading 3 is not a human-friendly aria-label so this is something that we might need to fix in another ticket

Reply:
Added a comment!

I don't think we have control over the label though, that comes from Gutenberg. It refers to the control in the block sidebar: 
<img width="250" src="https://user-images.githubusercontent.com/5656702/129210314-637b6836-9d29-4a06-b616-a20eb613b805.png" />